From ecfb7fa566da92b37982b4494069a3d1fb0be110 Mon Sep 17 00:00:00 2001
From: Dong Aisheng <b29396@freescale.com>
Date: Thu, 4 Jun 2015 17:03:57 +0800
Subject: [PATCH] MLK-11031-1 dts: fix dtb name of
 imx7d-12x12-lpddr3-arm2-mipi_dsi in Makefile

The *.dts file in dts/Makefile won't be compiled by make dtbs command.
Fix it.

Signed-off-by: Dong Aisheng <b29396@freescale.com>
---
 arch/arm/boot/dts/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index c88ebfb..c520b7b 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -228,7 +228,7 @@ dtb-$(CONFIG_ARCH_MXC) += \
 	imx7d-12x12-lpddr3-arm2-sim.dtb \
 	imx7d-12x12-lpddr3-arm2-m4.dtb \
 	imx7d-12x12-lpddr3-arm2-pcie.dtb \
-	imx7d-12x12-lpddr3-arm2-mipi_dsi.dts \
+	imx7d-12x12-lpddr3-arm2-mipi_dsi.dtb \
 	imx7d-19x19-lpddr3-arm2.dtb \
 	imx7d-19x19-ddr3-arm2.dtb \
 	imx7d-19x19-ddr3-arm2-csi.dtb \
-- 
1.8.0

